Jenkinsfile

```diff
@@ -0,0 +1,150 @@
+def DETAILS_URL="https://www.rti.com"
```

Comment:
@jpovedano @fgaranda what do you think about this URL? Maybe we should put the community portal URL or something like that?

This appears when you click on the "details" link when the action is launched:

![image](https://user-images.githubusercontent.com/10894592/121881793-014e1f80-cd10-11eb-9edb-b61df7e98e11.png)

Reply:
My idea was to define the GitHub App's URL as the details URL once it is created. So maybe I will update it in the future.

Reply:
The DETAILS_URL was changed to https://www.community.rti.com/